### PR TITLE
draft for /projects

### PR DESCRIPTION
**Pull request by kba**

### CODE REVIEW

File: projects.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+In addition, eight module projects develop prototype implementations for various workflow steps:
+
+## [MP1] Scalable Methods of Text and Structure Recognition for the Full-Text Digitization of Historical Prints, Part 1.B: Image Optimization
+
+The project “Skalierbare Verfahren der Text- und Strukturerkennung für die
+Volltextdigitalisierung historischer Drucke” has the goal of developing a
+complete OCR-Workflow for a high quality mass digitization of historical prints
+from the 17th-18th century. For each step in the workflow innovative methods
+should be made available as tools. Module 1.B: Bildoptimierung is the basis on
+which high quality OCR provided. For each optimization step there are a wide
```
Comment (wrznr) [inline projects.md]: Non-parseable sentence.

File: projects.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+variety of algorithms available, however not all of them are suitable to the
+specific challenges of this projects. On the basis of prior experience and
+work, the DFKI plans the identification, development and integration of
+suitable methods.
```
Comment (wrznr) [inline projects.md]: Ein Hohn.

File: projects.md [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+  * GEPRIS: http://gepris.dfg.de/gepris/projekt/394343055?language=en
+* GitHub: 
+  * https://github.com/syedsaqibbukhari/docanalysis
+
+## [MP2] Scalable Methods of Text and Structure Recognition for the Full-Text Digitization of Historical Prints, Part 2: Layout Analysis
+
+The project “Skalierbare Verfahren der Text- und Strukturerkennung für die
+Volltextdigitalisierung historischer Drucke” has the goal of developing a
+complete OCR-Workflow for a high quality mass digitization of historical prints
+from the 16th-18th century. For each step in the workflow innovative methods
```
Comment (wrznr) [inline projects.md]: Comma after fronted `PP` (i.e. after *workflow*) missing.

File: projects.md [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+should be made available as tools. Module 2: Layouterkennung ist next to OCR
```
Comment (wrznr) [inline projects.md]: *ist* -> *is*

File: projects.md [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+itself the most important step. It improves the OCR results directly, but also
+improves the general understanding of the digitized document by providing
+insights to the layout and relations between the document components. For each
+optimization step there are a wide variety of algorithms available, however not
```
Comment (wrznr) [inline projects.md]: Comma after *step*.

File: projects.md (hunk 1 above)
Comment (kba) [reply projects.md]: GEPRIS abstract...

Comment (wrznr) [reply projects.md]: Then, please add `[sic]`.

File: projects.md [hunk 6]
```diff
@@ -0,0 +1,227 @@
...
+* GitHub: 
+  * https://github.com/ocr-d-modul-2-segmentierung/segmentation-runner
+
+## [MP5] Optimized use of OCR methods – Tesseract as a component of the OCR-D workflow
+
+Tesseract is a free software for text recognition (optical character
+recognition, OCR). This software has a history of more than 30 years of
+continuous development and improvements. In the small group of open source
+products for OCR Tesseract belongs to the programs with the best recognition
+rates.Since end of 2016 Tesseract supports state-of-the-art text recognition by
```
Comment (wrznr) [inline projects.md]: Missing space.

File: projects.md [hunk 7]
```diff
@@ -0,0 +1,227 @@
...
+neural networks (LSTM).The context of OCR-D requires well defined interfaces
```
Comment (wrznr) [inline projects.md]: Missing space.

File: projects.md [hunk 8]
```diff
@@ -0,0 +1,227 @@
...
+  * GEPRIS: http://gepris.dfg.de/gepris/projekt/393215159?language=en
+* GitHub: 
+  * https://github.com/cisocrgroup/ocrd-postcorrection
+
+## [MP7] Development of a Repository for OCR Models and an Automatic Font Recognition tool in OCR-D
+
+The project addresses the problem of strongly fluctuating recognition rates of
+OCR for 16th to 18th century historical prints, limiting the full-text
+digitization of material created by the VD16, VD17, and VD18
+programs.Recognition models trained on modern corpora lacking the specifics of
```
Comment (wrznr) [inline projects.md]: Missing space.

File: projects.md [hunk 9]
```diff
@@ -0,0 +1,227 @@
...
+historical prints or historic material without thorough bibliographic analysis,
+retard recognition rates in comparison to the accuracy now routinely achieved
+for scans of modern prints.The creation of font-specific corpora on the basis
```
Comment (wrznr) [inline projects.md]: Missing space.

File: projects.md [hunk 10]
```diff
@@ -0,0 +1,227 @@
...
+of manual tagging is unrealistic, since both non-trivial knowledge of printing
+history is necessary and the scalability of such an approach would be
+insufficient. Due to the repetitiveness of the task, such an approach is also
+very error-prone. The project will enable the humanities to use OCR in a
+font-specific manner with limited effort. In order to achieve this the project
+has three main objectives:The development of an online training infrastructure
```
Comment (wrznr) [inline projects.md]: Missing space.

File: projects.md [hunk 11]
```diff
@@ -0,0 +1,227 @@
...
+that allows specific models to be trained for these font groups and at the same
+time for different OCR software.Development of a tool for the automatic
```
Comment (wrznr) [inline projects.md]: Missing space.

File: projects.md [hunk 12]
```diff
@@ -0,0 +1,227 @@
...
+recognition of fonts in digitizations of historical prints. In this case, an
+algorithm for the recognition of fonts in incunabula is first trained using the
+ground truth found in the Typenrepertorium der Wiegendrucke. In a second step
+the fonts are grouped according to their similarity in order to get as few
+groups as possible while maintaining OCR accuracy.Provision of a model
```
Comment (wrznr) [inline projects.md]: Missing space.

File: projects.md (hunk 10 above)
Comment (wrznr) [reply projects.md]: @kba This one is still missing.

**Pull request merged**